lib/beaker-hostgenerator/data.rb

```diff
@@ -1139,6 +1139,7 @@ def generate_osinfo
       %w[almalinux rocky].each do |os|
         (8..9).each do |release|
           yield ["#{os}#{release}-64", "el-#{release}-x86_64"]
+          yield ["#{os}#{release}-AARCH64", "#{os}#{release}-aarch64"]
```

Comment:
I'm not really sure how I can verify that this is correct

Reply:
I think we'll want to stay consistent between x86_64 and AARCH64 in terms of using the EL as the platform for Alma and Rocky. Could you update to 

```
          yield ["#{os}#{release}-AARCH64", "el-#{release}-aarch64"]
```

Reply:
made the changes.